src/site/content/en/lighthouse-pwa/apple-touch-icon/index.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: Does not provide a valid apple-touch-icon
+description: |
+  How to specify what icon should appear on iOS homescreens for your Progressive Web App.
+web_lighthouse:
+  - apple-touch-icon
+---
+
+iOS Safari users can manually add Progressive Web Apps to their homescreens.
+You can specify what icon should appear on iOS homescreens by adding a 
+`<link rel="apple-touch-icon" href="/example.png">` tag to the `<head>` of your
```

Comment:
```suggestion
`<link rel="apple-touch-icon" href="/example.png">` tag in the `<head>` of your
```
Needed to change the preposition to "in" to go with "including," which I chose to differentiate what you, the developer, is doing from what the iOS user is doing (adding to the home screen).